scripts/lib/CIME/XML/grids.py

```diff
@@ -250,8 +251,9 @@ def _get_domains_v2(self, component_grids, atmlevregex, lndlevregex):
             domain_node = self.get_optional_node(nodename="domain", attributes={"name":grid_name_nonlev})
             if domain_node is not None:
                 comp_name = grid[0].upper()
-                domains[comp_name + "_NX"] = int(self.get_element_text("nx", root=domain_node))
-                domains[comp_name + "_NY"] = int(self.get_element_text("ny", root=domain_node))
+                if not comp_name == "MASK":
```

Comment:
This is related to the PR MASK_NX and MASK_NY were being set but are not variables that we ever define.